src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -2304,4 +2428,67 @@ public UIDispatcherEventArgs(Action a)
         /// </summary>
         public Action ActionToDispatch { get; set; }
     }
+
+    /// <summary>
+    /// The LacingStrategyConverter is used to serialize and deserialize LacingStrategy enum values.
+    /// The mapping to string like 'applyDisabled' is to support the historical representation
+    /// of 'lacing' on Flood.
+    /// </summary>
+    public class LacingStrategyConverter : JsonConverter
```

Comment:
API Change : Converter classes are added to support serialization and deserialization.